csrankings-d.csv

```diff
@@ -408,6 +408,7 @@ Danielle Azar,Lebanese American University,http://sas.lau.edu.lb/csm/people/dani
 Danielle Szafir,University of Colorado Boulder,http://www.colorado.edu/cmci/people/information-science/danielle-albers-szafir,CGb88B0AAAAJ
 Danilo Ardagna,Politecnico di Milano,http://home.deib.polimi.it/ardagna,2S6m-oYAAAAJ
 Danilo Avola,Sapienza University of Rome,http://visionlab.di.uniroma1.it/node/9,9R03Bl0AAAAJ
+Danilo Francati,Royal Holloway University of London,https://dfrancati.github.io,eeuQ0a0AAAAJ

```

Comment:
Not clearly in CS: https://www.royalholloway.ac.uk/research-and-teaching/departments-and-schools/computer-science/about-us/

Reply:
Apologies for not including a justification. Danilo holds a PhD in Computer Science from Stevens Institute. At RHUL, he delivers the module Security Testing for the Department of Computer Science on their MSci programme. Furthermore, he can solely advise students for a PhD in Computer Science as cross-departmental supervision in our EPMS School is common practice.